File: PartnerApi/Swift/bz_delete_computer.swift
import Foundation

let apiUrl = ""  // Provided by b2_authorize_account
let accountAuthorizationToken = ""  // Provided by b2_authorize_account
let accountId = ""  // Provided by b2_authorize_account
let computerId = ""  // Provided by bz_list_computers

let postData: [String: Any] = [
  "accountId": accountId,
  "computerId": computerId,
]

var request = URLRequest(url: URL(string: "\(apiUrl)/api/backup/v1/bz_delete_computer")!)
request.httpMethod = "POST"
request.setValue(accountAuthorizationToken, forHTTPHeaderField: "Authorization")
request.setValue("application/json", forHTTPHeaderField: "Content-Type")
request.httpBody = try? JSONSerialization.data(withJSONObject: postData)

let (data, response) = try await URLSession.shared.data(for: request)

print(String(decoding: data, as: UTF8.self))

File: PartnerApi/PHP/bz_delete_computer.php
<?php

$api_url = ''; // Provided by b2_authorize_account
$auth_token = ''; // Provided by b2_authorize_account
$account_id = ''; // Provided by b2_list_group_members
$computer_id = ''; // Provided by bz_list_computers

$postData = json_encode([
    'accountId' => $account_id,
    'computerId' => $computer_id,
]);

$ch = curl_init($api_url . '/api/backup/v1/bz_delete_computer');
curl_setopt_array($ch, [
    CURLOPT_RETURNTRANSFER => true,
    CURLOPT_POST           => true,
    CURLOPT_HTTPHEADER     => [
        "Authorization: $auth_token",
        'Content-Type: application/json',
    ],
    CURLOPT_POSTFIELDS     => $postData,
]);

echo curl_exec($ch), PHP_EOL;
